pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "submit_cgap"
-version = "0.8.0"
+version = "0.8.1"
```

Comment:
This PR adds a new feature, so the version should be 0.9.0 under semantic versioning rules.